src/options.js

```diff
@@ -25,7 +25,7 @@ class Options {
       return;
     }
 
-    if (!isNaN(+value)) {
+    if (value && !isNaN(+value)) {
```

Comment:
@RolKau can we move this to a setter for the port property only? [Just like the `pluginOrder` option](https://github.com/livereload/livereload-js/blob/40ddaf40456268bfbd5b6ecc2cd30c20122f6a01/src/options.js#L17).
This is quite a tricky if-statement if used for all (current and future) properties.